gobblin-runtime/src/test/java/org/apache/gobblin/runtime/spec_store/MysqlBaseSpecStoreTest.java

```diff
@@ -116,6 +136,22 @@ public void testAddSpec() throws Exception {
     Assert.assertFalse(this.specStore.exists(URI.create("dummy")));
   }
 
+  @Test
+  public void testUpdateSpec() throws Exception {
+    this.specStore.addSpec(this.flowSpec);
+    Assert.assertEquals(this.specStore.getSize(), 1);
+    Assert.assertTrue(this.specStore.exists(this.uri1));
+    //Updating
+
+    this.specStore.addSpec(this.flowSpecUpdated);
```

Comment:
given that `this.specStore` is a `MysqlBaseSpecStore` (on L78), I don't believe it actually uses the insert statement from `MysqlSpecStore` that you modified.

I'm open, but it would probably be cleanest if you made another test class specifically for that one.

when you do, the way to verify the "indexing columns" your new code is meaning to update would be to call `getSpecImpl(SpecSearchObject)`

Reply:
thanks,addressed